content/questions/actions/question-109.md

```diff
@@ -0,0 +1,13 @@
+---
+question: "What is the purpose of the `timeout-minutes` keyword in a step?"
+archetype: "questions"
+title: "Question 109"
+draft: false
+---
+
+
+> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes
```

Comment:
This is the one for steps, you linked the section for job.timeout-minutes
```suggestion
> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes
```

Reply:
updated